OpenRA.Game/Actor.cs

```diff
@@ -138,6 +139,35 @@ internal Actor(World world, string name, TypeDictionary initDict)
 			SyncHashes = TraitsImplementing<ISync>().Select(sync => new SyncHash(sync)).ToArray();
 		}
 
+		public void Created()
```

Comment:
If we make this internal it will guarantee that mod code can't call it. We can then drop the `InvalidOperationException` as being unnecessary.